packages/theme-patternfly-org/pages/global-css-variables.md

```diff
@@ -52,6 +83,22 @@ Breakpoint suffixes are used in utility classes and layouts as well as in many c
 
 PatternFly 4 styles provide a default starting point. You can use the variable system to make adjustments to that default styling. When you change one or more elements, you should package those values into a new SCSS stylesheet to replace the default styling.
 
+Overrides to PatternFly variables should be made at the `:root` level for global variables or at the top-level component selector for component variables (for example, `.pf-c-button`), as these overrides will cascade down to children elements accordingly.
+
+Example:
+- Change the global primary color to red, but keep the original primary blue `#06c` color as the background for primary buttons.
```

Comment:
I don't know that you need this, and not sure if it's confusing it's referred to as a hex value here and a css var below? You could also say it's helpful to be a little more detailed, and show that you should use a CSS var instead of a hex value. 

```suggestion
- Change the global primary color to red, but keep the original primary blue color as the background for primary buttons.
```